packages/client/test/rpc/engine/forkchoiceUpdatedV1.spec.ts

```diff
@@ -93,6 +93,26 @@ tape(`${method}: call with valid fork choice state without payload attributes`,
   await baseRequest(t, server, req, 200, expectRes)
 })
 
+tape(`${method}: invalid terminal block`, async (t) => {
+  const genesisWithHigherTtd = {
+    ...genesisJSON,
+    config: {
+      ...genesisJSON.config,
+      terminalTotalDifficulty: 17179869185,
+    },
+  }
+
+  const { server } = await setupChain(genesisWithHigherTtd, 'post-merge', {
+    engine: true,
+  })
+
+  const req = params(method, [validForkChoiceState, null])
+  const expectRes = (res: any) => {
+    t.equal(res.body.result.payloadStatus.status, 'INVALID_TERMINAL_BLOCK')
```

Comment:
thanks for adding a test! can you also add one to newPayload so we can get coverage on the error added that as well

Reply:
done :-D